### ISSUE TITLE
Support Self-Signed SSL

### ISSUE THREAD

**Issue by djcenox** [SocialEngine/rancher-cron#13]
If you use self-signed SSL to use rancher throught SSL, you get certification authority error when rancher-cron try to reach metadata. 
It should add ca.crt from /var/lib/rancher/etc/ssl/ca.crt same used by rancher agent itself.

**Comment by smazurov** [comment]
Do you mean [this](https://github.com/rancher/agent/blob/0607d1cb4f34f148688cb528bb60202bfbe176d2/utilities/docker/launch_unix.go#L27-L45)? I would imagine you still need to add cert yourself via a volume.

**Comment by djcenox** [comment]
Yes, you have to add the CA on that path. I guess the agent would load that path automatically and run update-ca-certs to add ca inside OS boundle. Anyway the cron isn't aware of that. I guess you could add an entrypoint to update certificates before running CMD.

**Comment by djcenox** [comment]
I think this is the line that add CA.crt to agent [here](https://github.com/rancher/rancher/blob/master/agent/run.sh#L7). Running as --privileged can get that crt easily.

**Comment by smazurov** [comment]
Ok. To summarize, we should pass an env var, that if present, loads those options into the client init. 

Then, add to readme instructions on how to create a volume and set that env var in case of self-signed cert. 

I don't currently have time to work on this, but PRs are welcome. It should be a relatively straightforward change.
